syntax = "proto3";

import "google/protobuf/any.proto";

message IntPB {
  bool int = 1;
  int32 value = 2;
}

message FloatPB {
  bool float = 1;
  float value = 2;
}

message StringPB {
  bool string = 1;
  string value = 2;
}

message BoolPB {
  bool bool = 1;
  bool value = 2;
}

message IterablePB {
  bool iter = 1;
  repeated google.protobuf.Any values = 2;
}

message MapPB {
  bool map = 1;
  map<string, google.protobuf.Any> items = 2;
}

message NonePB {
  bool none = 1;
}